src/defrag.c

```diff
@@ -1240,6 +1305,7 @@ static void beginDefragCycle(void) {
         if (dbHasNoKeys(dbid)) continue;
         addDefragStage(defragStageDbKeys, (void *)(uintptr_t)dbid, NULL);
         addDefragStage(defragStageExpiresKvstore, (void *)(uintptr_t)dbid, NULL);
+        addDefragStage(defragStageKeysWithvolaItemsKvstore, (void *)(uintptr_t)dbid, NULL);
```

Comment:
Note what's actually happening with the expires.  The stage is added only to defrag the expiration table (overhead) - NOT the keys referenced by that table.  Note that the `scanHashtableCallbackCountScanned` callback is used, which doesn't do anything.  With the old `dict`, this is equivalent to defragging only the `dictEntry` (but not the referenced `robj`).  Similar work is done in the new hashtable struct.

All of the real defragmentation (of robjs) was already done by `defragStageDbKeys`.